README.md

```diff
@@ -1,2 +1,58 @@
 # koa-npm-resolution
-Koa middleware that transforms NPM package specifiers to relative paths
+
+This is a middleware for Koa servers that resolves NPM package specifiers in ES modules to relative paths for use on the web.  It is currently only intended for use as middleware in development servers to fascilitate build-free testing/iteration.
```

Comment:
I'd put an example of an import and what it's rewritten to right after the first sentence

Reply:
Moved some stuff around and included Al's great suggestions into the doc.